api/main.py

```diff
@@ -101,6 +101,33 @@ async def get_node(node_id: str):
     return await db.find_by_id(Node, node_id)
 
 
+@app.get('/nodes')
+async def nodes():
+    return {'nodes': await db.find_all(Node)}
```

Comment:
I think we may add something like that later, with pagination and range parameters to only get a subset of all the objects in the returned value.  It's probably better to leave it out for now.

Reply:
In fact this probably does the same as the handler above with no query parameters.  I think we should drop this, what do you think?

Reply:
Yes, I tested it. We can drop this.